driver-core/src/main/java/com/datastax/driver/core/ColumnDefinitions.java

```diff
@@ -307,6 +308,23 @@ void checkType(int i, DataType.Name name) {
         return defined.getName();
     }
 
+    CodecRegistry getCodecRegistry() {
+        if(codecRegistry == null)
```

Comment:
should we make codecRegistry an AtomicReference to prevent possibility of returning two separate instances to concurrent callers, or is that not much of a worry?